rust/actyx/swarm/harness/src/bin/versions.rs

```diff
@@ -50,14 +50,7 @@ mod versions {
                         .to_owned(),
                 ))
             } else {
-                let arch = match Arch::current() {
-                    Arch::x86_64 => "amd64",
-                    Arch::aarch64 => "arm64",
-                    Arch::arm => "arm",
-                    Arch::armv7 => "armhf",
-                    x => panic!("unsupported arch: {}", x),
-                };
-                let name = format!("actyx-{}-linux-{}", version, arch);
+                let name = format!("actyx-{}-linux-{}", version, ARCH);
```

Comment:
This needs the same mapping as above; I think this should be provided in `version.rs`.

Reply:
Yeah, I'm still finishing this, I need to push from time to time so I get the code on the Hetzner boz